MobileCenterUpdates/MobileCenterUpdates/MSUpdates.h

```diff
@@ -3,7 +3,7 @@
 @interface MSUpdates : MSServiceAbstract
```

Comment:
Actually I didn't realized but it can't be customized since it has to be unique per app ^^